### PR TITLE
Shifting the dark/light toggle button in mobile view 

### LINKED ISSUE #403: Shift the dark/light toggle button in footer in mobile view

### PR DESCRIPTION
**Pull request by anushruti11**
#### Describe the changes you've made
Shifted the dark/light toggle button in footer in mobile view

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

#### Deploy your changes to github page or netlify and mention deployment link:
https://anushruti11.github.io/Voters-Registration-Portal/index.html

#### Test plan (OPTIONAL)
A good test plan should give instructions that someone else can easily follow.
How someone can test your code?

#### Checklist

- [x] I have mentioned related issue number in PR description.
- [ ] I have added screenshots and related demo links of my changes..
- [x] The title of my pull request is a short description of the requested changes.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] Open Source Program names(OPTIONAL: If you participated in any open-source program then please mention the program name here)

@nlok5923 check this out.

### CODE REVIEW

**Comment by nlok5923** [comment]
@anushruti11 thanks for opening this pr i am investigating on this issue in the mean time you can opt some other issues to work upon.

File: login.html [hunk 1]
```diff
@@ -440,11 +440,13 @@ <h4>Integrity Pledge
       </label>
      
  
-
+      <div class="col-12 col-sm-6 col-md-12">
```
Comment (nlok5923) [inline login.html:443]: just remove all this boostrap styling. you can use media query with 768 px break points

**Review by nlok5923 — CHANGES_REQUESTED** [review]
Thanks PTAL!!

File: login.html (hunk 1 above)
Comment (anushruti11) [reply login.html:443]: I tried using it for the footer-down class , it's not working.

Comment (nlok5923) [reply login.html:443]: don't use it for `footer-down` class instead try it using `labelnew` class also first you should try your changes on less code pages like `help.html` Once succeed try to implement same in all pages.

Comment (anushruti11) [reply login.html:443]: Ok I will let you know after implementing it  I guess both the code pages are almost same.

Comment (anushruti11) [reply login.html:443]: @nlok5923  for labelnew class media query with 768 breakpoints is already implemented .Please check it once in the style.css file.

Comment (nlok5923) [reply login.html:443]: yeah that's what i am saying try to manipulate position in that media query the styles mentioned in that media query are not correct and that's the point of this issue. As previously we had the toggle switch on top so may be the current styling in that media query is according to previous toggle button position (button in header) you need to fix that only via considering the newer changes (which is toggle button in footer).

**Comment by anushruti11** [comment]
@QAZIMAAZARSHAD I have made all the changes please review it now.

**Comment by QAZIMAAZARSHAD** [comment]
Check login page.

**Comment by anushruti11** [comment]
> Check login page.

image

**Comment by QAZIMAAZARSHAD** [comment]
image

**Comment by QAZIMAAZARSHAD** [comment]
Compare with other pages.

**Comment by anushruti11** [comment]
> Compare with other pages.

Both login and sign up page have common CSS file. I haven't applied anything different for login page.

**Comment by QAZIMAAZARSHAD** [comment]
Something must be wrong.
In desktop view also login page has a slightly different position of the toggle button.


Review code again.

**Comment by anushruti11** [comment]
> Something must be wrong.
> In desktop view also login page has a slightly different position of the toggle button.
> 
> Review code again.

Added separate styling for login page in  the login.html file
image

File: login.html [hunk 2]
```diff
@@ -470,11 +527,13 @@ <h4>Integrity Pledge
...
      
  
-
+    
   <div class="copyright" style="padding-bottom:20px">
     Copyright ©
     <script>document.write(new Date().getFullYear())</script> Qazi Maaz Arshad
     </div>
+  
+
```
Comment (nlok5923) [inline login.html:536]: remove this unncessary new lines

**Review by nlok5923 — CHANGES_REQUESTED** [review]
@anushruti11 also can you please make remove the scrolling effect once user click on toggle button

**Comment by nlok5923** [comment]
code and UI both looks good to me just remove the scroll effect.

**Comment by anushruti11** [comment]
> code and UI both looks good to me just remove the scroll effect.

I haven't added the scroll effect it was there in the original website. How it can be removed?

**Comment by nlok5923** [comment]
> > code and UI both looks good to me just remove the scroll effect.
> 
> I haven't added the scroll effect it was there in the original website. How it can be removed?

Yeah I am aware of that but it's fine if you're not willing to do it in this pr will create another issue for the same @QAZIMAAZARSHAD the changes looks good to me just after resolving recent requested changes I think we can merge the pr.

**Comment by anushruti11** [comment]
> > > code and UI both looks good to me just remove the scroll effect.
> > 
> > 
> > I haven't added the scroll effect it was there in the original website. How it can be removed?
> 
> Yeah I am aware of that but it's fine if you're not willing to do it in this pr will create another issue for the same @QAZIMAAZARSHAD the changes looks good to me just after resolving recent requested changes I think we can merge the pr.

@nlok5923 Thanks for guiding me..

**Pull request merged**